examples/scroll/lv_example_scroll_1.c

```diff
@@ -1,15 +1,53 @@
 #include "../lv_examples.h"
 #if LV_BUILD_EXAMPLES
 
+lv_obj_t  * panel;
+lv_obj_t  * save_button;
+lv_obj_t  * restore_button;
+int         saved_scroll_x;
+int         saved_scroll_y;
```

Comment:
```suggestion
static lv_obj_t  * panel;
static lv_obj_t  * save_button;
static lv_obj_t  * restore_button;
static int         saved_scroll_x;
static int         saved_scroll_y;
```

Normally this is harmless but if a user has any global `panel` in their own code and they compile LVGL examples they will get a linker error.